nle/env/base.py

```diff
@@ -186,8 +186,7 @@ class StepStatus(enum.IntEnum):
 
     def __init__(
         self,
-        savedir="",
-        archivefile="nethack.%(pid)i.%(time)s.zip",
```

Comment:
After talking to Eric he convinced me to leave this here for now in order to not confuse participants too much.

Let's just remove savedir in this PR (and also the archivefile=None calls above, given that the argument gets ignored) and we'll deal with archivefile in a future PR.

Reply:
Can archivefile be defaulted to None, too, then? Same argument as for savedir

Reply:
Yes.